<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>LiNuXT</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-LiNuXT-Desktop-Debug/Unbenannt.png</normaloff>../build-LiNuXT-Desktop-Debug/Unbenannt.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="tabStopWidth">
       <number>30</number>
      </property>
      <property name="cursorWidth">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2">
     <widget class="QTreeView" name="treeView">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionChose_File"/>
    <addaction name="actionCreate_new_File"/>
    <addaction name="actionSave_File"/>
    <addaction name="separator"/>
    <addaction name="actionFinish"/>
   </widget>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionCompile_Program"/>
    <addaction name="actionDownload_Program_to_the_NXT"/>
    <addaction name="actionDownload_Program_and_run_it"/>
    <addaction name="separator"/>
    <addaction name="actionEnhanced_Firmware"/>
   </widget>
   <widget class="QMenu" name="menuCompiler">
    <property name="title">
     <string>Compiler</string>
    </property>
    <addaction name="actionUse_root_You_have_to_start_the_application_with_root_rights_2"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionFind"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuArduino">
    <property name="title">
     <string>Arduino</string>
    </property>
    <addaction name="actionUse_the_Arduino_Language_to_Program_your_NXT"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuProgram"/>
   <addaction name="menuCompiler"/>
   <addaction name="menuArduino"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionChose_File"/>
   <addaction name="actionSave_File"/>
   <addaction name="actionCreate_new_File"/>
   <addaction name="separator"/>
   <addaction name="actionCompile_Program"/>
   <addaction name="actionDownload_Program_to_the_NXT"/>
   <addaction name="actionDownload_Program_and_run_it"/>
  </widget>
  <action name="actionInstall_Compiler">
   <property name="text">
    <string>Install Compiler</string>
   </property>
  </action>
  <action name="actionFinish">
   <property name="text">
    <string>Finish</string>
   </property>
  </action>
  <action name="actionCompile_Program">
   <property name="icon">
    <iconset theme="tools-check-spelling">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Compile Program</string>
   </property>
  </action>
  <action name="actionDownload_Program_to_the_NXT">
   <property name="icon">
    <iconset theme="emblem-downloads">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Download Program to the NXT</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionChose_File">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCreate_new_File">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionInstall">
   <property name="text">
    <string>Install</string>
   </property>
  </action>
  <action name="actionUSB0">
   <property name="text">
    <string>USB0</string>
   </property>
  </action>
  <action name="actionUSB1">
   <property name="text">
    <string>USB1</string>
   </property>
  </action>
  <action name="actionUSB2">
   <property name="text">
    <string>USB2</string>
   </property>
  </action>
  <action name="actionUSB3">
   <property name="text">
    <string>USB3</string>
   </property>
  </action>
  <action name="actionUSB4">
   <property name="text">
    <string>USB4</string>
   </property>
  </action>
  <action name="actionUSB5">
   <property name="text">
    <string>USB5</string>
   </property>
  </action>
  <action name="actionUSB6">
   <property name="text">
    <string>USB6</string>
   </property>
  </action>
  <action name="actionUSB7">
   <property name="text">
    <string>USB7</string>
   </property>
  </action>
  <action name="actionUSB8">
   <property name="text">
    <string>USB8</string>
   </property>
  </action>
  <action name="actionUSB9">
   <property name="text">
    <string>USB9</string>
   </property>
  </action>
  <action name="actionDownload_Program_and_run_it">
   <property name="icon">
    <iconset theme="system-run">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Download Program and run it</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionEnhanced_Firmware">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enhanced Firmware</string>
   </property>
  </action>
  <action name="actionUse_root_You_have_to_start_the_application_with_root_rights_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use root (You have to start the application with root rights)</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUse_the_Arduino_Language_to_Program_your_NXT">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use the Arduino Language to Program your NXT</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
